framework/src/modules/token/method.ts

```diff
@@ -753,7 +753,7 @@ export class TokenMethod extends BaseMethod {
 			throw new Error('All tokens are supported.');
 		}
 
```

Comment:
Comment about `isTokenSupported` function (line 669): According to[ LIP specs](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0051.md#istokensupported), this function should also check if `tokenID == getTokenIDLSK()`. I am not sure that the implementation does this check, would be good to verify.

Reply:
yes, it's checking in the [store](https://github.com/LiskHQ/lisk-sdk/blob/release/6.0.0/framework/src/modules/token/stores/supported_tokens.ts#L57)

Reply:
ah I see. Just to make sure I understand: The corresponding check is done in the `isMainchainOrNative` function; [there](https://github.com/LiskHQ/lisk-sdk/blob/1e9af209ac7a055e73e4de210dc7c3bec41feca0/framework/src/modules/token/stores/supported_tokens.ts#L188C17-L188C30), instead of calling precisely `getTokenIDLSK()`, you just (essentially) re-write the same code, right?

Reply:
yea i think so